SwiftUI_Homework/SwiftUI_Homework/ActivityView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ActivityView.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 20.01.2023.
+//
+
+import SwiftUI
+
+/// Интеграция ActivityView в SwiftUI
```

Comment:
```suggestion
/// Активити вью
```
По коду и так понятно